lib/modules/manager/sbt/extract.ts

```diff
@@ -12,7 +12,9 @@ import type { PackageDependency, PackageFile } from '../types';
 import type { ParseContext, ParseOptions } from './types';
 
 const stripComment = (str: string): string =>
-  str.replace(regEx(/(^|\s+)\/\/.*$/), '');
+  str
+    .replace(regEx(/(^|\s+)\/\/.*$/), '')
+    .replace(regEx(/\/(\*)+.*(\*)+\//), ''); /** allow comment */
```

Comment:
i think this will be to greedy.

```
/** t */
value
/** c */
```

would be completely removed

Reply:
This invoke be `extractPackageFile` in line 355
It actually run 1 line.